qmk_porting/keyboards/imk64/keymaps/jactry/keymap.c

```diff
@@ -29,7 +29,11 @@ tap_dance_action_t tap_dance_actions[] = {
 };
 
 /* Some customize keys. */
-#define MY_LSFT TD(TD_LSFT_CAPS)
+#define CK_LSFT TD(TD_LSFT_CAPS)
+
+enum custom_keycodes {
+    CK_OSMD = SAFE_RANGE,
```

Comment:
With reference to [this](https://github.com/O-H-M2/qmk_port_ch582/blob/via/qmk_porting/protocol/extra_keycode.h#L51), you may consider using `NEW_SAFE_RANGE` instead.
```suggestion
    CK_OSMD = NEW_SAFE_RANGE,
```